packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -49,6 +51,8 @@ interface EmulatorProcessInfo {
 export class AndroidEmulatorDevice extends DeviceBase {
   private emulatorProcess: ChildProcess | undefined;
   private serial: string | undefined;
+  private nativeLogsOutputChannel: OutputChannel | undefined;
+  private nativeLogsCancelToken = new CancelToken();
```

Comment:
I think one cancel token shouldn't be cancelled more than once. We should create new token in `mirrorLogs` method